packages/server-wallet/jest/chain-setup.ts

```diff
@@ -6,21 +6,13 @@ import {deploy} from '../deployment/deploy';
 
 export default async function setup(): Promise<void> {
   process.env['CHAIN_NETWORK_ID'] = '0x01';
-  process.env['GANACHE_HOST'] = '0.0.0.0';
-  process.env['GANACHE_PORT'] = '8545';
-  process.env[
-    'RPC_ENDPOINT'
-  ] = `http://${process.env['GANACHE_HOST']}:${process.env['GANACHE_PORT']}`;
 
   const accounts = ETHERLIME_ACCOUNTS.map(account => ({
     ...account,
     amount: utils.parseEther('100').toString(),
   }));
 
-  if (!process.env.GANACHE_PORT) {
-    throw new Error('process.env.GANACHE_PORT must be defined');
-  }
-  const ganacheServer = new GanacheServer(parseInt(process.env.GANACHE_PORT), 1337, accounts);
+  const ganacheServer = new GanacheServer(8545, 1337, accounts);
```

Comment:
Have you tried changing these magic constants to ensure that this is the only place that relies these? Specifically, other places our code might rely on `process.env.GANACHE_PORT` and might default to port 8545. If that were the case, then changing 8545 to 8546 would result in failing tests.